netty/src/test/java/io/grpc/netty/TlsTest.java

```diff
@@ -176,6 +178,70 @@ public void basicClientServerIntegrationTest() throws Exception {
     client.unaryRpc(SimpleRequest.getDefaultInstance());
   }
 
+  /**
+   * Same as basicClientServerIntegrationTest except we test the InputStream based certs API.
+   */
+  @Test
+  public void basicClientServerIntegrationTestInputStreamCerts() throws Exception {
+    InputStream serverCertChain = null;
```

Comment:
None of these InputStreams are passed to our code. This test is not adding any value.

Reply:
These assigned to on lines 193/194 , 218/219 and used immediately after.

Reply:
I see, I will remove the test.